controllers/contact-controller.js

```diff
@@ -171,7 +204,8 @@ function isValidContact(contact) {
     !isNullOrEmptyObject(contact.rawJson.callType) &&
     !isNullOrEmptyObject(contact.rawJson.childInformation) &&
     !isNullOrEmptyObject(contact.rawJson.callerInformation) &&
-    !isNullOrEmptyObject(contact.rawJson.caseInformation)
+    !isNullOrEmptyObject(contact.rawJson.caseInformation) &&
+    nonDataOrSummary(contact)
```

Comment:
Probably the "correct" approach for managing data quality in this kind of situation is to:
a) Keep a version number of the schema within the JSON here (e.g, `{"version": "2020-04-15"}`).  This allows us to change schemas over time and for different helplines.
b) define a [JSON schema](https://json-schema.org/) and validate against that schema when the request comes in.  This will ensure data integrity and we can be confident that the fields are as we expect.

I realize that's maybe too focused on implementation.  The point is that we should have a way to ensure data integrity, and need to support multiple versions of the schema over time.  And... none of this needs to be handled now.  I've created a Jira issue to track this.